contributing.md

```diff
@@ -0,0 +1,40 @@
+## To run microsite 
+
+## Steps for raising a Pull Request
+1. Please raise an issue in [here](https://github.com/zio/zio-config/issues) before you raise a pull request directly.
+2. Tag the issue in PR description.
+3. Contributions to documentation/website is always welcome and can be raised without tagging an issue.
+
+## SBT
+
+```scala
+cd zio-config
+sbt +compile
+sbt runAllExamples
```

Comment:
`PropertiesFileExample.scala` is broken.

Reply:
Sure. I will fix it

Reply:
@senia-psm  This was fixed ok